app/screens/receive-bitcoin-screen/receive-btc.tsx

```diff
@@ -448,6 +449,12 @@ const ReceiveBtc = () => {
     )
   }
 
+  if (state === PaymentRequestState.Paid) {
```

Comment:
we can also check for payment failure.
```suggestion
  if (state === PaymentRequestState.Paid || PaymentRequestState.Error) {
```

Reply:
Okay! added